knowledge_base/job_read_secret/databricks.yml

```
@@ -0,0 +1,47 @@
+bundle:
+  name: job-read-secret-example
+
+# workspace:
+#   host: https://myworkspace.cloud.databricks.com
+
+resources:
+  secret_scopes:
+    my_secret_scope:
+      name: secrets-scope-1
```

```suggestion
      name: prod-secrets-scope
```

nit: having prod in the name is more descriptive.